Lib/test/test_binascii.py

```diff
@@ -241,6 +242,15 @@ def test_crc32(self):
 
         self.assertRaises(TypeError, binascii.crc32)
 
+    def test_random_crc32(self):
+        dat = random.randbytes(1234)
```

Comment:
Tests that run on random data have a good chance of being flakey when a problem exists rather than reliably reproducing the problem. The specific data or the seed used to generate it needs to be recorded and present in test output to meaningfully debug the problem.

The flakiness aspect can be improved by using a lot of runs over different random data (statistically likely to always trigger the problem being tested for on at least one random input) or by using a smaller set of strategically chosen inputs with desirable output values to test against.

For crc32 I suggest just chosing a few strategic inputs that generate low and high crc32 values that might have illustrated this issue and need for the & mask in Python 2.

Reply:
> For crc32 I suggest just chosing a few strategic inputs that generate low and high crc32 values that might have illustrated this issue and need for the & mask in Python 2.

I ran an `alder32()`/`crc32()` test code, tested nearly 2 billion random 1KiB strings, no one result is greater than `UINT32_MAX`, although the type of return value is `uLong` (unsigned long).

Reply:
I remove the unit-tests, since no code actually changed.